packages/react-strict-dom/tests/html/html-test.native.js

```diff
@@ -392,6 +392,25 @@ describe('<html.*> (native polyfills)', () => {
         });
       });
 
+      test('"autoCapitalize" prop', () => {
+        [
+          // web only
+          'on',
+          'off',
+          // web & native
+          'none', // used instead of 'off'
+          'sentences', // used instead of 'on'
+          'words',
+          'characters'
```

Comment:
This test is just for native, and if we're supporting all 6 values it's confusing (and inaccurate) to have comments like "web only".